curation_concerns-models/app/models/concerns/curation_concerns/file_set/derivatives.rb

```diff
@@ -16,11 +17,15 @@ def create_derivatives(filename)
         when *self.class.pdf_mime_types
           Hydra::Derivatives::PdfDerivatives.create(filename,
                                                     outputs: [{ label: :thumbnail, format: 'jpg', size: '338x493', url: derivative_url('thumbnail') }])
+          Hydra::Derivatives::FullTextExtract.create(filename,
+                                                     outputs: [{ url: uri, container: "extracted_text" }])
         when *self.class.office_document_mime_types
           Hydra::Derivatives::DocumentDerivatives.create(filename,
                                                          outputs: [{ label: :thumbnail, format: 'jpg',
                                                                      size: '200x150>',
                                                                      url: derivative_url('thumbnail') }])
+          Hydra::Derivatives::FullTextExtract.create(filename,
+                                                     outputs: [{ url: uri, container: "extracted_text" }])
         when *self.class.audio_mime_types
           Hydra::Derivatives::AudioDerivatives.create(filename,
                                                       outputs: [{ label: 'mp3', format: 'mp3', url: derivative_url('mp3') },
```

Comment:
Why are we only extracting text from PDF and office doc types? Tika can extract from *lots* of kinds of files: http://tika.apache.org/1.10/formats.html

Do the other types present particular problems we're working around?